### DISCUSSION TITLE
High amount of queries

### DISCUSSION THREAD

**Discussion by pureflo** [dgreif/ring#867] [Q&A] [answered]
I was wondering if this is now a normal thing happening within the Ring plugin for HomeBridge or it's just the Ring doorbell itself.

192.168.1.5 = HomeBridge and PiHole.
192.168.1.1 = Router

[pi-hole will forward unknown DNS requests to 192.168.1.1 which in turn will forward to CloudFlare DNS]

Jan 29 14:05:50 dnsmasq[12131]: query[A] api.ring.com from 192.168.1.5
Jan 29 14:05:50 dnsmasq[12131]: forwarded api.ring.com to 192.168.1.1
Jan 29 14:05:50 dnsmasq[12131]: query[AAAA] api.ring.com from 192.168.1.5
Jan 29 14:05:50 dnsmasq[12131]: forwarded api.ring.com to 192.168.1.1
Jan 29 14:05:50 dnsmasq[12131]: validation result is INSECURE
Jan 29 14:05:50 dnsmasq[12131]: reply api.ring.com is <CNAME>
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 34.198.242.237
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 54.145.98.124
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 52.200.138.148
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 34.199.1.171
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 52.202.84.53
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 34.227.180.193
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 107.23.113.100
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is 52.45.163.161
Jan 29 14:05:50 dnsmasq[12131]: validation result is INSECURE
Jan 29 14:05:50 dnsmasq[12131]: reply api.ring.com is <CNAME>
Jan 29 14:05:50 dnsmasq[12131]: reply clientsapigw.ring.com is NODATA-IPv6

Jan 29 14:05:53 dnsmasq[12131]: query[A] api.ring.com from 192.168.1.5
Jan 29 14:05:53 dnsmasq[12131]: forwarded api.ring.com to 192.168.1.1
Jan 29 14:05:53 dnsmasq[12131]: query[AAAA] api.ring.com from 192.168.1.5
Jan 29 14:05:53 dnsmasq[12131]: forwarded api.ring.com to 192.168.1.1
Jan 29 14:05:53 dnsmasq[12131]: validation result is INSECURE
Jan 29 14:05:53 dnsmasq[12131]: reply api.ring.com is <CNAME>
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is NODATA-IPv6
Jan 29 14:05:53 dnsmasq[12131]: validation result is INSECURE
Jan 29 14:05:53 dnsmasq[12131]: reply api.ring.com is <CNAME>
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 34.237.30.30
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 75.101.151.51
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 52.72.108.124
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 52.203.170.70
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 34.198.192.245
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 3.226.216.208
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 34.195.80.59
Jan 29 14:05:53 dnsmasq[12131]: reply clientsapigw.ring.com is 54.81.216.224

As you can see, it's happening every 3 seconds.

Any thoughts/suggestions would be greatly appreciated.

Thanks.

**Comment by dgreif** [comment, marked as answer]
We have to poll the Ring api for "dings" to know when motion and doorbell presses occur. That's why you see these so often
